### PR TITLE
Adding linux to the wv2 roadmap page

### LINKED ISSUE #1638: No mention of Linux in the roadmap?
[Enter feedback here]
Why is there no mention of Linux in the roadmap?  Has that been dropped?  If so, WebView2 is almost worthless from my company's viewpoint since we support both Windows and Linux in many of our products.  If it were easy, we'd love to support MacOS as well but have limited resources.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: dec166ea-3192-50ac-a930-f2185226801f
* Version Independent ID: a7055048-1b57-b2dc-78ae-6786d8765467
* Content: [Roadmap for Microsoft Edge WebView 2 - Microsoft Edge Development](https://docs.microsoft.com/en-us/microsoft-edge/webview2/roadmap)
* Content Source: [microsoft-edge/webview2/roadmap.md](https://github.com/MicrosoftDocs/edge-developer/blob/main/microsoft-edge/webview2/roadmap.md)
* Product: **microsoft-edge**
* Technology: **webview**
* GitHub Login: @MSEdgeTeam
* Microsoft Alias: **msedgedevrel**

### PR DESCRIPTION
**Pull request by captainbrosset**
The WebView2 team is planning to work on Linux support sometime next year. This is being tracked here: https://github.com/MicrosoftEdge/WebView2Feedback/issues/645

However the roadmap doc does not mention it, and people have been asking about it.

Fixes #1638.

**Rendered article for review:**
https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/roadmap?branch=pr-en-us-1642
Added the Linux list item in intro section.
Docs team added a full Writer/Editor pass on the article.

### CODE REVIEW

**Comment by vchapel** [comment]
Why is the first line "Last Updated: July 2021". I cannot find this line in any of our docs.

File: microsoft-edge/webview2/roadmap.md [hunk 1]
```diff
@@ -54,4 +55,4 @@ The .NET SDK has reached GA.
 
 ### Windows UI Library 3
```
Comment (mikehoffms) [inline microsoft-edge/webview2/roadmap.md:56]: This section is in the GA section.  Has this technology reached GA, or just preview?

File: microsoft-edge/webview2/roadmap.md [hunk 2]
```diff
@@ -1,48 +1,49 @@
...
 ms.technology: webview
 keywords: IWebView2, IWebView2WebView, webview2, webview, win32 apps, win32, edge, ICoreWebView2, ICoreWebView2Host, browser control, edge html
+ms.date: 01/07/2021
 ---
-# Microsoft Edge WebView2 roadmap
+# WebView2 roadmap
 
-> [!NOTE]
-> Last Updated:  July 2021
+Last Updated: July 2021
```
Comment (Reezaali) [inline microsoft-edge/webview2/roadmap.md:14]: Since we have the MSDate at the top of the article, can we remove this one?

**Review by mikehoffms — APPROVED** [review]

File: microsoft-edge/webview2/roadmap.md (hunk 2 above)
Comment (captainbrosset) [reply microsoft-edge/webview2/roadmap.md:14]: We probably should. I'll remove it now.

**Review by jasonstephen15 — APPROVED** [review]

**Pull request merged**